src/sentry/demo/data_population.py

```diff
@@ -77,6 +78,30 @@
 
 logger = logging.getLogger(__name__)
 
+contexts_by_platform = {
```

Comment:
@roggenkemper can we call this `contexts_by_mobile_platform`?

Reply:
yes we can and we will!